### PR TITLE
Added new rune, added nether npc, removed some alpha items

### PR DESCRIPTION
**Pull request by jani270**

### CODE REVIEW

File: items/UNIQUE_RUNE.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "§7Creates a captivating Enchanting",
+    "§7Halo! Obtained from Fire Sales.",
+    "",
+    "§7Apply this rune to helmet or",
+    "§7fuse two together at the Runic",
+    "§7Pedestal!",
+    "",
+    "§6§lLEGENDARY COSMETIC"
+  ],
+  "internalname": "UNIQUE_RUNE",
```
Comment (romangraef) [inline items/UNIQUE_RUNE.json:19]: SPELLBOUND_UNIQUE_RUNE :pleading_face:

Comment (jani270) [reply items/UNIQUE_RUNE.json:19]: No

Comment (romangraef) [reply items/UNIQUE_RUNE.json:19]: you will regret this

**Review by NopoTheGamer — APPROVED** [review]
image

**Review by Lulonaut — APPROVED** [review]

**Pull request merged**